routers/api/v1/notify/notifications.go

```diff
@@ -19,15 +19,15 @@ func NewAvailable(ctx *context.APIContext) {
 	// summary: Check if unread notifications exist
 	// responses:
 	//   "200":
-	//    "$ref": "#/responses/NotificationCount"
+	//     "$ref": "#/responses/NotificationCount"
 	//   "204":
-	//     description: No unread notification
+	//     "$ref": "#/responses/NotificationCount"
 
 	count := models.CountUnread(ctx.User)
 
 	if count > 0 {
 		ctx.JSON(http.StatusOK, api.NotificationCount{New: count})
 	} else {
-		ctx.Status(http.StatusNoContent)
+		ctx.JSON(http.StatusNoContent, api.NotificationCount{New: count})
```

Comment:
`204 No Content` prohibits the generation of a body.

https://tools.ietf.org/html/rfc7231#section-6.3.5

>    A 204 response is terminated by the first empty line after the header
>   fields because it cannot contain a message body.

Reply:
@guillep2k I could use 404 but this is an error

Reply:
Use 200